### DISCUSSION TITLE
I want an &quot;install&quot; URL protocol

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#104] [Wants]
The idea of boookmarking/installing a website is pretty confusing for most users. Instead of teaching users how to use bookmark/install in their browsers we should add an &quot;install&quot; protocol for URLs (like `tel:`, `sms:`, etc.). This way developers can add a link to their site, with `href=&quot;install:yourwebsite.com&quot;` (or similar) - and if the users clicks on it the page will be bookmarked/installed (if the browser supports it), or the browser will ask what app to use (similar to what they do with `tel:` now if you click it in a desktop browser with no app available to handle `tel:` links).

As a next step, marketing could use these as well, adding &quot;install:yourwebsite.com&quot; to their ads and other marketing materials to make it obvious that the website can be installed like a &quot;native app&quot;.

<hr>

https://webwewant.fyi/wants/81/

**Comment by ghost** [comment]
@aarongustafson this idea is wonderful, amazing idea

**Comment by 62f** [comment]
[POC](install-as-app://www.youtube.com/shorts/fPjbt1aH40Q)
I guess one day when the protocol is implemented, the above hyperlink will work. Atm it doesn't even light up in gh's Preview tab.
